packages/lab/src/slider/Slider.css

```diff
@@ -1,201 +1,109 @@
-  --slider-handle-outlineStyle: var(--saltSlider-handle-outlineStyle, var(--salt-focused-outlineStyle));
-  --slider-handle-outlineWidth: var(--saltSlider-handle-outlineWidth, var(--salt-focused-outlineWidth));
-  --slider-handle-outlineColor: var(--saltSlider-handle-outlineColor, var(--salt-focused-outlineColor));
-  --slider-handle-outlineOffset: var(--saltSlider-handle-outlineOffset, var(--salt-focused-outlineOffset));
-
-  --slider-arrow-height: var(--saltSlider-arrow-height, 6px);
-  --slider-arrow-width: var(--saltSlider-arrow-width, 8px);
-  --slider-arrow-color: var(--saltSlider-arrow-color, var(--slider-selection-color));
-
-  --slider-borderStyle: var(--saltSlider-borderStyle, none);
-  --slider-borderWidth: var(--saltSlider-borderWidth, 0);
-  --slider-borderColor: var(--saltSlider-borderColor, transparent);
-  --slider-width: var(--saltSlider-width, 300px);
-
-  --slider-clickable-paddingLeft: var(--saltSlider-clickable-paddingLeft, calc(var(--salt-size-base) * 0.5));
-  --slider-clickable-paddingRight: var(--saltSlider-clickable-paddingRight, calc(var(--salt-size-base) * 0.5));
-
-  --slider-label-fontSize: var(--saltSlider-label-fontSize, var(--salt-text-label-fontSize));
-  --slider-label-paddingLeft: var(--saltSlider-label-paddingLeft, 0);
-  --slider-label-paddingRight: var(--saltSlider-label-paddingRight, 0);
+.saltSlider-labelMaxBottom {
+  grid-row: 2;
+  grid-column: 1;
+  justify-self: end;
+  width: min-content;
 }
 
-.saltSlider {
-  width: var(--slider-width);
-  border-style: var(--slider-borderStyle);
-  border-width: var(--slider-borderWidth);
-  border-color: var(--slider-borderColor);
-
+.saltSliderTrack {
   display: flex;
-  flex-direction: column;
-  align-items: stretch;
-
-  padding-top: var(--slider-paddingTop);
-  padding-bottom: var(--slider-paddingBottom);
+  position: relative;
+  cursor: pointer;
+  align-items: center;
 }
 
-.saltSlider-disabled {
+.saltSliderTrack-rail {
+  height: var(--salt-size-bar);
+  background: var(--salt-track-borderColor);
+  width: 100%;
+  position: absolute;
 }
 
-.saltSlider-clickable {
-  padding: var(--slider-clickable-paddingTop) var(--slider-clickable-paddingRight) var(--slider-clickable-paddingBottom) var(--slider-clickable-paddingLeft);
-  margin-right: calc(-1 * var(--slider-clickable-paddingRight));
-  margin-left: calc(-1 * var(--slider-clickable-paddingLeft));
+.saltSliderTrack:active {
+  cursor: grabbing;
+}
 
+.saltSliderThumb-container {
+  position: absolute;
   display: flex;
-  flex-direction: column;
+  align-items: center;
   justify-content: center;
-  align-items: stretch;
+  transform: translate(-50%, 0%);
+  height: 50px;
+  width: 50px;
 }
 
-.saltSlider-track {
-  display: grid;
-  grid-template-rows: auto auto auto;
-  align-items: end;
-  grid-template-columns: auto auto auto;
-  row-gap: 0;
-  transition: grid-template-columns 100ms ease;
+.saltSliderThumb-tooltip {
```

Comment:
This meant to mimic the Salt Tooltip so should this include the tooltip overrides, like --saltTooltip-XXX ?

Reply:
Don't worry about it